messages.md

```diff
@@ -14,8 +14,8 @@ channel](network.md).
 ## Watchtower
 
 At least one watchtower is ran by every stakeholder, and at most one wallet is paired with
-each watchtower. A watchtower needs to be able to sign any revocation transaction before
-its corresponding wallet signs the unvaulting transaction.
+each watchtower. A watchtower needs to be able to see all fully-signed revocation transaction
```

Comment:
If tasked with the emergency response, a watchtower needs to be able to see all fully-signed revocation transactions before its corresponding wallet signs the unvaulting transaction. Otherwise, a watchtower needs to be able to see the fully-signed cancel transaction before its corresponding wallet signs the unvaulting transaction.

Reply:
This document is only about hosted watchtowers, so i don't think it should be a concern. We really don't have specified the communication with external WTs